### PR TITLE
SP-8269-add-client-method

### PR DESCRIPTION
**Pull request by bartoszflis-silvair**
mergetrain: after #53

### CODE REVIEW

File: bluetooth_mesh/models/models.py [hunk 1]
```diff
@@ -1581,6 +1581,29 @@ async def get_lightness(
             for node, result in results.items()
         }
 
+    async def set_lightness_range_unack(
```
Comment (michallowasrzechonek-silvair) [inline bluetooth_mesh/models/models.py:1584]: `_range_`?

Comment (bartoszflis-silvair) [reply bluetooth_mesh/models/models.py:1584]: Aaah, copy paste error.

File: bluetooth_mesh/models/models.py [hunk 2]
```diff
@@ -1581,6 +1581,29 @@ async def get_lightness(
...
+            *,
+            retransmissions: int = 6,
+            send_interval: float = 0.075,
+    ) -> None:
+        async def request():
+            ret = self.send_app(
+                destination,
+                app_index=app_index,
+                opcode=LightLightnessOpcode.LIGHTNESS_SET_UNACKNOWLEDGED
+                params=dict(lightness=lightness)
```
Comment (michallowasrzechonek-silvair) [inline bluetooth_mesh/models/models.py:1598]: Hm, what about delay? These are application retries, so let's try to avoid the popcorn

Comment (michallowasrzechonek-silvair) [reply bluetooth_mesh/models/models.py:1598]: Also, I think the event might include transition time

Comment (bartoszflis-silvair) [reply bluetooth_mesh/models/models.py]: I've just added it.

Comment (bartoszflis-silvair) [reply bluetooth_mesh/models/models.py]: Fixed delay handling as well.

**Review by michallowasrzechonek-silvair — APPROVED** [review]

**Pull request merged**